features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/site/conf/locales/jaggery/locale_default.json

```diff
@@ -342,6 +344,7 @@
     "No documentation associated with the API": "No documentation associated with the API",
     "No matching APIs found.": "No matching APIs found.",
     "No uploaded certificates for the endpoints": "No uploaded certificates for the endpoints",
+    "No uploaded client certificates for the API": "No uploaded client certificates for the API",
```

Comment:
"No client certificates were found uploaded for this API." or its even better if we can just say "No client certificate found for this API. Please upload a certificate"